bin/genesis

```diff
@@ -4872,19 +4865,17 @@ sub prompt_for_vault {
 			my $env = $envs{$_}{env};
 			explain "#Y{Determining Vault paths needed for $env ...}\n";
 
-			my $dir = workdir;
-			if (is_create_env($env)) {
-				write_stemcell_data("$dir/cloud.yml"); # Why write stemcell to cloud.yml??!!
```

Comment:
Because we merge cloud.yml into the kit files, and the proto BOSH feature flag activates kit YAML fragments that pull meta.latest_stemcell based on IaaS type.